api/api/spec/spec.yaml

```diff
@@ -208,43 +247,32 @@ components:
       - pending
       - neverconnected
       - disconnected
+      description: Agent status. It is calculated based on the last keep alive and the Wazuh version.
+
+    AgentID:
+      type: string
+      minLength: 3
+      description: Agent ID.
 
     AgentConfiguration:
       type: object
-      required:
-        - configuration
+      description: Configuration the agent is currently using. The output of this API call depends on the component requested and the agent configuration.
+
+    CDBList:
+      type: object
       properties:
-        configuration:
+        items:
```

Comment:
Better add this `items` property in the definition of the response in order to keep clean the CDBList schema.

Reply:
Hello @crd1985,

Inside an CDB List object there's an item property:
```javascript
# curl -u foo:bar "localhost:55000/lists?pretty&limit=1&offset=3"
{
   "error": 0,
   "data": {
      "totalItems": 1,
      "items": [
         {
            "path": "etc/lists/audit-keys",
            "items": [
               {
                  "key": "audit-wazuh-w",
                  "value": "write"
               },
               {
                  "key": "audit-wazuh-r",
                  "value": "read"
               },
               {
                  "key": "audit-wazuh-a",
                  "value": "attribute"
               },
               {
                  "key": "audit-wazuh-x",
                  "value": "execute"
               },
               {
                  "key": "audit-wazuh-c",
                  "value": "command"
               }
            ]
         }
      ]
   }
}
```
The model is correct.